modules/virt-connecting-to-vm-console-web.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+ifeval::["{context}" == "vnc-console"]
+:vnc-console:
+:console: VNC console
+:console-menu: VNC console
+endif::[]
+ifeval::["{context}" == "serial-console"]
+:serial-console:
+:console: serial console
+:console-menu: Serial console
+endif::[]
+ifeval::["{context}" == "desktop-viewer"]
+:desktop-viewer:
+:console: desktop viewer
+:console-menu: Desktop viewer
+endif::[]
+
+:_content-type: PROCEDURE
+[id="virt-connecting-to-vm-console-web_{context}"]
+ifdef::vnc-console,serial-console[]
+= Using the web console to connect to the {console}
+
+You can use the {product-title} web console to connect to the {console} of a virtual machine (VM).
+endif::[]
+ifdef::desktop-viewer[]
+= Using the web console to configure the {console}
+
+You can use the {product-title} web console to configure the {console} to access a Windows virtual machine (VM).
+endif::[]
+
+ifdef::vnc-console[]
+[NOTE]
+====
+If you connect to a Windows VM with a vGPU assigned as a mediated device, you can switch between the default display and the vGPU display.
+====
+endif::[]
+
+ifdef::desktop-viewer[]
+.Prerequisites
+
+* QEMU guest agent is installed on the Windows VM. The QEMU guest agent is included in the VirtIO drivers.
+* An RDP client is installed on a machine on the same network as the Windows VM.
```

Comment:
Is this still true when an RDP service is used?

Reply:
I don't know. I could not set up a Windows VM, so I did not test RDP. But how else would a user open a remote viewer without an RDP client?

@dominikholler  Do you know whether an RDP client is required if an RDP service is used?

Reply:
> I could not set up a Windows VM

applying https://gitlab.cee.redhat.com/dholler/example-vms/-/blob/main/windows11.yaml will give you one.

> Is this still true when an RDP service is used?

@0xFelix I do not understand your point, do you refer to "QEMU guest agent is installed on the Windows VM. The QEMU guest agent is included in the VirtIO drivers." ?

Reply:
I meant that when a service is created the network requirements might be different. The RDP client machine doesn't have to be on the same network?

Reply:
@dominikholler @0xFelix 
This is the only comment that has not been resolved. Could someone check whether the RDP client machine needs to be on the same network as the Windows VM?

Reply:
@0xFelix  fits
```suggestion
* You have an RDP client installed.
```
better?

Reply:
When using a service it doesn't have to be on the same network.